benchmarks/bigint_conversion.py

```diff
@@ -0,0 +1,98 @@
+import argparse
```

Comment:
Note, needs to be added to `run_benchmarks` to get auto-run in nightly.

Reply:
good catch!